src/tailwind-sx/src/sxGenerator.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+import csstree from 'css-tree';
+import * as changeCase from 'change-case';
+
+const NOT_SUPPORTED_PROPS = new Set([
+  'grid-gap',
+  'grid-row-gap',
+  'grid-column-gap',
+  'pointer-events',
+]);
+const UNITLESS_PROPS = new Set(['line-height', 'font-size']);
+
+// eslint-disable-next-line flowtype/no-weak-types
+export default function generateSX(css: string | Buffer): Object {
+  const ast = csstree.parse(css);
+  const sx = {};
+
+  csstree.walk(ast, {
+    enter(node) {
+      if (node.type === 'ClassSelector') {
+        if (this.atrule != null) {
+          return;
+        }
+
+        const pseudoClassSelector = getPseudoSelectorName(this.rule);
+        const className = node.name.replace('\\', '');
```

Comment:
This GitHub code scanning warning is worth checking (not sure if it's valid or if you saw it) - see below this comment.